pkgs/development/libraries/tiledb/default.nix

```diff
@@ -11,6 +11,7 @@
 , openssl
 , boost
 , libpqxx
+, git
```

Comment:
Why is this now required?

Reply:
Otherwise the build fails like this:

```
-- Adding Magic as an external project
CMake Error at /nix/store/vnhl4zdy7igx9gd3q1d548vwzz15a9ma-cmake-3.27.7/share/cmake-3.27/Modules/ExternalProject.cmake:2851 (message):
  error: could not find git for clone of ep_magic
Call Stack (most recent call first):
  /nix/store/vnhl4zdy7igx9gd3q1d548vwzz15a9ma-cmake-3.27.7/share/cmake-3.27/Modules/ExternalProject.cmake:4345 (_ep_add_download_command)
  cmake/Modules/FindMagic_EP.cmake:90 (ExternalProject_Add)
  cmake/TileDB-Superbuild.cmake:92 (include)
  CMakeLists.txt:190 (include)
```